gpii/node_modules/ontologyHandler/src/ontologyHandler.js

```diff
@@ -106,6 +107,111 @@ URLs - this aspect is taken care of in the preferences server component.
             transformSpecs[filename] = JSON.parse(transformSpec);
         });
         that.options.ontologyTransformSpecs = transformSpecs;
+        // Generate Ontology Routes
+        var edges = [];
+        fluid.each(transformSpecs, function(spec, key) {
+            // The transforms are bidirectional so we add both directed edges
+            edges.push(key.split('-'), key.split('-').reverse());
```

Comment:
Move the logic for inferring keys to a single shared function, so we can have this peculiar convention stored in once place